app/controllers/assessments_controller.rb

```diff
@@ -608,27 +608,25 @@ def viewFeedback
 
   def getPartialFeedback
     job_id = params["job_id"].to_i
-    resp = {}
+
     # User requested to view feedback on a score
     if job_id.nil?
       flash[:error] = "Invalid job id"
       redirect_to(action: "index") && return
     end
 
-    begin
+    resp = get_job_status(job_id)
+
+    if resp["is_assigned"]
       resp['partial_feedback'] = tango_get_partial_feedback(job_id)
-    rescue AutogradeError
-      # if unable to get partial feedback, check if job is on the queue
-      @job_status = get_job_status(job_id)
-      resp["is_assigned"] = @job_status["is_assigned"]
-      resp["queue_position"] = @job_status["queue_position"]
-      resp["queue_length"] = @job_status["queue_length"]
     end
 
     render json: resp.to_json
   end
 
   def parseScore(feedback)
+    return if feedback.nil?
```

Comment:
May I ask what is this for?

Reply:
in the error logs chaskiel sent, it looks like there's some cases where @score.feedback doesn't get properly set even when autograding is complete, which causes an error in this function.